### ISSUE TITLE
Hourly funding allows zero hours as input

### ISSUE THREAD

**Issue by jayalaves** [AutarkLabs/open-enterprise#1565]
**Describe the bug**
Funding new issues in hourly model allows me to input zero as value.

**Mainnet or testnet?**
Rinkeby

**Organization**
http://rinkeby.aragon.org/#/nov5oetest/

**To Reproduce**
1. Set funding model to Hourly
2. Fund new issue and insert zero hours
3. Submit

**Expected behavior**
Need tighter input validation around this value. It cannot be zero.

**Screenshots**
View [on video here](https://www.youtube.com/watch?v=RqUKq8-qrF4&feature=youtu.be&t=3598).

**Desktop (please complete the following information):**
 - OS and OS version: macOS Catalina 10.15
 - Browser and browser version: Google Chrome 78.0.3904.87 (Official Build) (64-bit)

**Comment by chadoh** [comment]
@jayalaves this was addressed by #1637 

One question, though. The error message doesn't quite match the rest of the form, since it talks about "funding amount" while the input is called "hours". Also, it displays an error as soon as the form is loaded, rather than waiting until after a focus+blur of the input. Do these things need to change, or is it good enough as-is?

<img width="287" alt="funding amount must be greater than zero" src="https://user-images.githubusercontent.com/221614/69073179-e19f2a00-09fa-11ea-91c9-30e168a5384b.png">

**Comment by jayalaves** [comment]
@chadoh I can't seem to reproduce your issue there, but funny enough in the process of trying to reproduce I think I uncovered a different issue.

<img width="1440" alt="Screenshot 2019-11-18 at 19 03 25" src="https://user-images.githubusercontent.com/28843778/69077603-4166f680-0a36-11ea-831a-707e63672715.png">

Before touching the estimated amount container, nothing looks unexpected.

<img width="1440" alt="Screenshot 2019-11-18 at 19 03 31" src="https://user-images.githubusercontent.com/28843778/69077633-517ed600-0a36-11ea-9c36-6305368ca461.png">

Inserting a zero gives me an expected error message.

<img width="1440" alt="Screenshot 2019-11-18 at 19 03 46" src="https://user-images.githubusercontent.com/28843778/69077668-60fe1f00-0a36-11ea-885d-f8976559ec8f.png">

Leaving the input container empty _after_  removing a previously inserted number gives me this error, which makes no sense to me in terms of what is it that this number means.

But to answer your question, yes, I agree that in this case the wording is slightly incorrect. It should be "Estimated hours must be greater than zero"

**Comment by jayalaves** [comment]
Referencing this other issue here #1656.

**Comment by chadoh** [comment]
@jayalaves sorry, you were looking at an old version of the code on Rinkeby, I believe. The errors you noticed were fixed by #1637, but maybe your DAO never got updated to the latest version of the frontend.

Anyhow, the point about the wording of the message still stands. I'll change that. Do you also want me to make sure that the error only shows up after at least one input has been focused-then-unfocused, and is still zero or blank?

**Comment by jayalaves** [comment]
@chadoh Yes, display the error only after having tried to insert zero or blank. Otherwise, no need to display the error beforehand.
